tutorials/fundamentals/undefined-check.md

```diff
@@ -0,0 +1,29 @@
+In JavaScript, checking if a variable is undefined can be a bit tricky since
+a [null](/tutorials/fundamentals/null) variable can pass a check for undefined if not written properly.
+As a result, this allows for `undefined` values to slip through and vice versa.
+This is because `null` and `undefined` both mean nothing, so checking if they
+are equal `==` would return true. However, if you were to check if they were
+strictly equal `===`, it would return false as they are different ways of saying nothing.
+
+```javascript
+[require:Fundamentals good-undefined-check]
+```
+
+Objects are special because over the course of a program execution, properties
```

Comment:
I'd move this into a separate section entitled "Checking Whether Object Properties are Undefined" and explain the difference between checking whether `obj.x === undefined` versus checking `obj.hasOwnProperty(x)`: https://masteringjs.io/tutorials/fundamentals/hasownproperty